spec/test-data/translations/ios_generate_strings_file_from_code/sample-project/Pods/SomePod/Sources/PodClass1.swift

```diff
@@ -1,5 +1,9 @@
 import Foundation
 
+func PodLocalizedString(_ key: String, value: String? = nil, comment: StaticString) -> String {
+    Bundle(for: BundleToken.self).localizedString(forKey: key, value: value, table: nil)
+}
```

Comment:
Note that the real use case in client apps will most likely be **Widgets** having a `AppLocalizedString` helper method to reference strings from the host app, rather than us parsing methods like this from **Pods**.